### PR TITLE
Create jan-01.md

### PR DESCRIPTION
**Pull request by xtuc**
We should we do the meeting?

### CODE REVIEW

**Comment by xtuc** [comment]
Please select your availabilities in https://doodle.com/poll/zwhc2f4dmnn4yazk, this one is open to public.

File: 2018-01/jan-0.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- Sven: idea about using babel-preset-env in the root configuration and transpile deps against the target
+
+## Plugin order
+
+- Still priority? probably not a blocker for the 7 release
+
+## Async plugins/visitors
+
+- Not for now
+- Would explose if Babel supports async for plugins or visitor
```
Comment (loganfsmyth) [inline 2018-01/jan-0.md]: `expose`.

File: 2018-01/jan-0.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## async-to-generator
```
Comment (loganfsmyth) [inline 2018-01/jan-0.md]: `async-to-promise`?

File: 2018-01/jan-0.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+## Babel Weekly Notes - 2018-01-01
+
+## Participant
+
+- Logan Smyth
+- Nicolo Ribaudo
```
Comment (nicolo-ribaudo) [inline 2018-01/jan-0.md]: Nicolò (with the accent 😛)

File: 2018-01/jan-0.md (hunk 1 above)
Comment (nicolo-ribaudo) [reply 2018-01/jan-0.md]: Maybe also add a line about babel-register being a big blocker for this?

File: 2018-01/jan-0.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+- Should we keep the two syntaxes?
+- Could people opt-in into the syntax/old syntax?
+- We can publish a single transformation (with `legacy: true`)
+- Plugin ordering issue, we could handle oursevles
+
+## gsoc/rgsoc
+
+- We had probably too many people last year
+- We should assign our members to specific areas and mentor studient
+- Maybe 2 studients and 4 of our members to actually mentors
```
Comment (nicolo-ribaudo) [inline 2018-01/jan-0.md]: `students`

File: 2018-01/jan-0.md (hunk 2 above)
Comment (xtuc) [reply 2018-01/jan-0.md]: Yes!

File: 2018-01/jan-0.md (hunk 3 above)
Comment (xtuc) [reply 2018-01/jan-0.md]: Yes I didn't know how to type it lol

**Comment by xtuc** [comment]
Thanks guys!

File: 2018-01/jan-0.md (hunk 2 above)
Comment (xtuc) [reply 2018-01/jan-0.md]: done in https://github.com/babel/notes/commit/dbfff35a3ecf1278935269504a04c23a90052922

**Pull request merged**